CHANGELOG.md

```diff
@@ -5,3 +5,4 @@
 - Fixes Storage Emulator rules resource evaluation (#4214).
 - Fixes bug where securityLevel is overwritten on https function re-deploys (#4208).
 - Fixes bug where functions emulator ignored functions.runtime option in firebase.json (#4207).
+- Fixes bug where functions emulator triggered wrong functions when when started w/ --inspect-functions flag (#4232).
```

Comment:
Typo: "wrong functions when when" should just be "wrong functions when"

Also prefer "with" instead of "w/"